server/routerlicious/packages/lambdas/src/foreman/lambda.ts

```diff
@@ -99,14 +100,13 @@ export class ForemanLambda implements core.IPartitionLambda {
                         type: "tasks:start",
                     },
                 );
-                this.context.log?.info(
-                    `Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`,
-                    {
-                        messageMetaData: {
-                            documentId: docId,
-                            tenantId,
-                        },
-                    });
+                Lumberjack.log(`Request to ${queueName}: ${clientId}:${JSON.stringify(tasks)}`, LogLevel.Info,
```

Comment:
I introduced some additional convenience methods in #7397. Can you use `Lumberjack.info()` here instead?